### ISSUE TITLE
SecurityException: not allowed to perform FINE_LOCATION

### ISSUE THREAD

**Issue by jbaginski** [MobileAppTracking/sdk-release#18]
tune-marketing-console-sdk v4.1.0

```
Fatal Exception: java.lang.SecurityException: xxx from uid 10125 not allowed to perform FINE_LOCATION
       at android.os.Parcel.readException(Parcel.java:1546)
       at android.os.Parcel.readException(Parcel.java:1499)
       at android.location.ILocationManager$Stub$Proxy.requestLocationUpdates(ILocationManager.java:582)
       at android.location.LocationManager.requestLocationUpdates(LocationManager.java:948)
       at android.location.LocationManager.requestLocationUpdates(LocationManager.java:496)
       at com.tune.location.TuneLocationListener$GetLocationUpdates.run(SourceFile:197)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:139)
       at android.app.ActivityThread.main(ActivityThread.java:5298)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:950)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:745)
```

**Comment by john-gu** [comment]
This issue is fixed in the v4.1.3 release, please try updating to this version.

**Comment by jbaginski** [comment]
thanks @john-gu
